src/ideascale/models/se.rs

```diff
@@ -0,0 +1,47 @@
+use serde::Serialize;
+
+#[derive(Debug, Serialize)]
+pub struct Challenge {
```

Comment:
It may become a burden to maintain two sets of structures.
Is it not possible to define `Serialize` and `Deserialize` over a single `Challenge` struct?

Reply:
I decided to use 2 because data to be deserialize and to be serialize is not the same. Not just the types but also the content itself. Some attributes are just needed to reconstruct part of the data of other structs, but may not appear in the final `de -> se` representation. Is much simpler this way instead of dealing with the serde `optional` and `skip_de/serialize` dance.